tools/rdm/TestDefinitions.py

```diff
@@ -5071,6 +5071,8 @@ def Test(self):
       self.SetNotRun('Unable to determine pin code')
       return
 
+    # Section 3.9, out of range pins return NR_FORMAT_ERROR rather than
+    # NR_DATA_OUT_OF_RANGE like one may expect.
```

Comment:
Not when NR_DATA_OUT_OF_RANGE is returned for an invalid PIN. I'm just thinking a message at the end of the comment saying NR_DATA_OUT_OF_RANGE is returned when an incorrect PIN is given.